README.rst

```diff
@@ -30,13 +30,13 @@ your system:
 
 .. code-block:: bash
 
-    $ wget http://get.sensiolabs.org/php-cs-fixer.phar -O php-cs-fixer
+    $ wget https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/download/v1.11.4/php-cs-fixer.phar -O php-cs-fixer
```

Comment:
Would be great if `http://get.sensiolabs.org/php-cs-fixer.phar` can automatically redirect to the latest stable version instead.

Like PHPUnit does: https://phar.phpunit.de/phpunit.phar

What do you think?